bluesky/plans.py

```diff
@@ -2691,6 +2691,148 @@ def inner_relative_adaptive_scan():
     return (yield from inner_relative_adaptive_scan())
 
 
+def tune_centroid(
+        detectors, signal, motor, 
+        start, stop, min_step, 
+        num_points = 10, 
```

Comment:
A trivial request: no spaces between keyword arguments and their default values for PEP8 compliance. I'd be happy to add that commit myself if you'd like.

Reply:
I'll get it.